webpackShims/ui-bootstrap.js

```diff
@@ -1,11 +1,27 @@
 define(function (require) {
```

Comment:
Due to loading the translations directly with angular-translate then the custom loader factory function is now obsolete.